include/openPMD/IO/ADIOS/ParallelADIOS1IOHandlerImpl.hpp

```diff
@@ -76,7 +76,7 @@ namespace openPMD
         void listAttributes(Writable*, Parameter< Operation::LIST_ATTS > &) override;
 
         virtual int64_t open_write(Writable *);
-        virtual ADIOS_FILE* open_read(std::string const& name);
+        virtual ADIOS_FILE* open_read(std::string name);
```

Comment:
This also just takes the ```.c_str()``` without sinking the string itself.

Reply:
yep, reverted this one